### DISCUSSION TITLE
The props "closeIcon" dosen't work when I use Modal.method() in V5

### DISCUSSION THREAD

**Discussion by wang-man** [ant-design/ant-design#44175] [General]
please tell me why and how to solve this problem

**Comment by kiner-tang** [comment]
It work for me： https://codesandbox.io/s/ji-ben-antd-5-8-2-forked-35s4xs?file=/demo.tsx
<img width="1428" alt="image" src="https://github.com/ant-design/ant-design/assets/10286961/e40010df-bb61-478d-b87a-ce8fc468e406">

**Reply by wang-man** [reply]
I tried,it works beside me now, tks, I can't found the 'closable' prop in doc of V5 but it's still available
